### PR TITLE
Checklist action changes

### PR DESCRIPTION
**Pull request by abdrahmandiab**

### CODE REVIEW

File: .github/workflows/checklist_actions.yml [hunk 1]
```diff
@@ -9,31 +9,28 @@ jobs:
...
-      - uses: actions/checkout@v2
-      - run: |
-          echo A comment on PR $NUMBER
-          echo The body of comment $ID is "$BODY"
-        env:
-          NUMBER: ${{ github.event.issue.number }}  
-          BODY: ${{ github.event.comment.body }}
-          ID: ${{ github.event.comment.id }}
+      - name: Checkout
+        uses: actions/checkout@v2
```
Comment (Aarsh2001) [inline .github/workflows/checklist_actions.yml:13]: I suggest you checkout to ivy like this -:
```
 uses: actions/checkout@v2
        with:
          path: ivy
          persist-credentials: false
```

and put `reformat_checklist.md` somewhere inside `automation tools`

File: .github/workflows/checklist_actions.yml [hunk 2]
```diff
@@ -9,31 +9,28 @@ jobs:
...
         id: category
         with:
           value: ${{ github.event.comment.body }}
           index_of_str: "add_reformatting_checklist_"
+
       - name: Render template
         id: template
         uses: chuhlomin/render-template@v1.4
         with:
           template: .github/reformat_checklist.md
```
Comment (Aarsh2001) [inline .github/workflows/checklist_actions.yml:26]: do `ivy/automation_tools/somewhere/reformat_checklist.md`

**Review by Aarsh2001 — CHANGES_REQUESTED** [review]
I suggest you look at the following changes, nothing too big. Just that it would align the way we do things and have them a little more organised in fashion

**Comment by abdrahmandiab** [comment]
> I suggest you look at the following changes, nothing too big. Just that it would align the way we do things and have them a little more organised in fashion

Hi Aarsh, I've made all the changes you requested :)
If you think it's all good now you can go ahead and merge or let me know and I'll do the merge myself 😃

**Review by Aarsh2001 — APPROVED** [review]

**Pull request merged**